src/Microsoft.Health.Fhir.TemplateManagement/Client/ACRClient.cs

```diff
@@ -19,31 +19,31 @@
 
 namespace Microsoft.Health.Fhir.TemplateManagement.Client
 {
-    public class ACRClient : IOCIArtifactClient
+    public class AcrClient : IOciClient
     {
         private readonly IAzureContainerRegistryClient _client;
 
-        public ACRClient(string registry, string token)
+        public AcrClient(string registry, string token)
         {
             EnsureArg.IsNotNull(registry, nameof(registry));
             EnsureArg.IsNotNull(token, nameof(token));
 
-            _client = new AzureContainerRegistryClient(registry, new ACRClientCredentials(token));
+            _client = new AzureContainerRegistryClient(registry, new AcrClientCredentials(token));
         }
 
-        public ACRClient(IAzureContainerRegistryClient client)
+        public AcrClient(IAzureContainerRegistryClient client)
         {
             EnsureArg.IsNotNull(client, nameof(client));
             _client = client;
         }
 
-        public async Task<Stream> PullBlobAsStreamAcync(string imageName, string digest, CancellationToken cancellationToken = default)
+        public async Task<Stream> PullBlobAsStreamAcync(string name, string digest, CancellationToken cancellationToken = default)
```

Comment:
This function seems only used in this class. Do we need to make it public?

Reply:
We can publish this api to support pulling blob as stream.